src/main/java/com/google/cloud/spanner/pgadapter/utils/Metrics.java

```diff
@@ -44,7 +44,7 @@ public static List<Double> getMetricLatencyMillisBuckets() {
     List<Double> rpcMillisBucketBoundaries = new ArrayList<Double>(MAX_NUM_FINITE_BUCKETS);
     for (int i = 0; i < MAX_NUM_FINITE_BUCKETS && bucketValue <= MAX_VALUE; i++) {
       rpcMillisBucketBoundaries.add(bucketValue);
-      bucketValue = SCALE_FACTOR * BASE;
```

Comment:
Isn't the more appropriate naming here the following:

```
final double BASE = 0.25;
final double SCALE_FACTOR = 1.25;
double bucketValue = BASE;
...
for ... {
  bucketValue *= SCALE_FACTOR;
}
```

Reply:
You're right. Updated.